dev/beamdev

```diff
@@ -126,6 +126,19 @@ function start {
     docker-compose up --no-build --no-recreate --abort-on-container-exit
 }
 
+function demo {
+    if [ "$(git rev-parse --abbrev-ref HEAD)" == "main" ]; then
+        export TAG=main
+    else
+        echo "You are checked out to a non main branch of beam. We will launch the latest develop version of beam."
```

Comment:
```suggestion
        echo "You have checked out a non-main branch of beam. We will launch the latest develop version of beam. Your working copy remains untouched. In case of any problems, please switch to a clean develop branch."
```